src/test/Components/ImagesTable/ImagesTable.test.js

```diff
@@ -401,12 +405,19 @@ describe('Images Table', () => {
 
       // render the expected <ImageLink /> and compare the text content for a link
       render(
-        <ImageLink
-          imageStatus={mockStatus[compose.id].image_status}
-          uploadOptions={
-            compose.request.image_requests[0].upload_request.options
-          }
-        />,
+        <BrowserRouter>
```

Comment:
Why include BrowserRouter

Reply:
`ImageLink` component uses `useNavigate` to process recreation of an image. When I've tried to run the tests without a router they kept failing with: `Error: useNavigate() may be used only in the context of a <Router> component.`

Reply:
oh wow, that must've been there for a while then :) Ok that's fine.